lib/assets/stylesheets/overrides.scss

```diff
@@ -477,15 +482,24 @@ $tooltip-background: $grey;
 }
 
 /* Plan creation */
-.create-plan-or {
-  padding: 5px 0 0 15px;
-}
-.create-plan-checkbox {
-  margin-left: -40px;
-  margin-top: -5px;
-}
-.create-plan-mock {
-  margin-left: -40px;
+#new_plan {
```

Comment:
I don't think we need these styles for tweaking the new_plan form elements. You could easily adjust the elements in the view using col-md-*, pull-left, pull-right, etc. Can you have a try, please?

Reply:
Spent an hour trying but its going to be more work than just swapping in some Bootstrap classes and reorganizing the row/col classes. This ticket is for adding Asterisks to required fields not refactoring the html/css of pages. You can open a separate ticket if you feel its important